crate_universe/bootstrap.bzl

```diff
@@ -88,11 +89,26 @@ _crate_universe_resolver_bootstrapping = repository_rule(
             allow_single_file = ["Cargo.toml"],
             default = Label("//crate_universe:Cargo.toml"),
         ),
+        "iso_date": attr.string(
+            doc = "The iso_date of cargo binary the resolver should use. Note `version` must be `beta` or `nightly`",
```

Comment:
```suggestion
            doc = "The iso_date of cargo binary the resolver should use. Note: This can only be set if `version` is `beta` or `nightly`",
```